### ISSUE TITLE
minify fails when minifying Bootstrap v4/Font Awesome v5 css

### ISSUE THREAD

**Issue by nero120** [coderaiser/minify#50]
Attempting to use minify with Bootstrap v4 or Font Awesome v5 results in an `undefined` result. The offending styles are listed below.

To reproduce, add each style into a file `test.css` and run cli command `minify test.css`. For the Font Awesome styles you'll need to download the [Font Awesome](https://www.npmjs.com/package/@fortawesome/fontawesome-free) npm package and copy the contents of the `webfonts` folder into the same location as the above `test.css` file.

```
// Bootstrap v4 forms break minify
.was-validated .form-control:invalid, .form-control.is-invalid {
  border-color: #dc3545;
  padding-right: calc(1.5em + 0.75rem);
  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' fill='%23dc3545' viewBox='-2 -2 7 7'%3e%3cpath stroke='%23dc3545' d='M0 0l3 3m0-3L0 3'/%3e%3ccircle r='.5'/%3e%3ccircle cx='3' r='.5'/%3e%3ccircle cy='3' r='.5'/%3e%3ccircle cx='3' cy='3' r='.5'/%3e%3c/svg%3E");
  background-repeat: no-repeat;
  background-position: center right calc(0.375em + 0.1875rem);
  background-size: calc(0.75em + 0.375rem) calc(0.75em + 0.375rem); }
```

```
// Font Awesome v5 eot breaks minify
@font-face {
  font-family: 'Font Awesome 5 Free';
  font-style: normal;
  font-weight: 900;
  src: url('#./fa-solid-900.eot'); }
```

```
// Font Awesome v5 svg breaks minify
@font-face {
  font-family: 'Font Awesome 5 Free';
  font-style: normal;
  font-weight: 900;
  src: url('#./fa-solid-900.svg#fontawesome') format('svg'); }
```

**Comment by coderaiser** [comment]
Thank you 🙂, the thing is [css-b64-images](https://github.com/Filirom1/css-base64-images) can't process such files, so they will be minified only without images bundling. Fixed with 415b9b8. Landed in [v4.1.1](https://github.com/coderaiser/minify/releases/tag/v4.1.1) 🎉.

If you interested no embedding images of such files in `base64` submit an issue in `css-base64-images` repo.

```sh
coderaiser@cloudcmd:~/minify$ minify 1.css
[ Error: Skip data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' fill='%23dc3545' viewBox='-2 -2 7 7'%3e%3cpath stroke='%23dc3545' d='M0 0l3 3m0-3L0 3'/%3e%3ccircle r='.5'/%3e%3ccircle cx='3' r='.5'/%3e%3ccircle cy='3' r='.5'/%3e%3ccircle cx='3' cy='3' r='.5'/%3e%3c/svg%3E External file.
      at base64img (/home/coderaiser/minify/node_modules/css-b64-images/lib/css-b64-images.js:49:17)
      at iterate (/home/coderaiser/minify/node_modules/css-b64-images/lib/css-b64-images.js:101:5)
      at forEachSeries (/home/coderaiser/minify/node_modules/css-b64-images/lib/css-b64-images.js:115:3)
      at fromString (/home/coderaiser/minify/node_modules/css-b64-images/lib/css-b64-images.js:42:3)
      at Promise (internal/util.js:290:30)
      at new Promise (<anonymous>)
      at fromString (internal/util.js:289:12)
      at module.exports (/home/coderaiser/minify/lib/img.js:29:12)
      at /home/coderaiser/minify/node_modules/try-to-catch/lib/try-to-catch.js:6:31 ]
```

Is it works for you?

**Comment by nero120** [comment]
Awesome work @coderaiser!
